tests/unit/handle-deprecation-workflow-test.js

```diff
@@ -87,6 +87,7 @@ module('handleDeprecationWorkflow', function (hooks) {
 
   test('deprecation silenced with string matcher', function (assert) {
     const config = {
+      throwOnUnhandled: true,
```

Comment:
These are unrelated to the actual PR, but I think these prevent false positives. The "silenced" test cases in particular don't assert anything specific (`assert.ok(true)` in L100), only make sure that nothing throws. But without the `throwOnUnhandled` flag that wouldn't happen anyways, with or without the explicit workflow rule.